cmd/node/config/enableEpochs.toml

```diff
@@ -191,6 +191,9 @@
     # all nodes from staking queue are moved in the auction list
     StakingV4InitEnableEpoch = 1000000
 
+    # StakingV4EnableEpoch represents the epoch when staking v4 is enabled. Should have a greater value than StakingV4InitEnableEpoch
+    StakingV4EnableEpoch = 1000001
```

Comment:
same comment, might put this to 3-4 along with the StakingV4InitEnableEpoch in order to be able to test this internally

Reply:
Right, changed epoch flag value to 5 (since StakingV4InitEnableEpoch = 4)